Added link to setting unit_of_measurement in history graph card. (#21366)
home-assistant/home-assistant.github.io: source/_lovelace/history-graph.markdown

Screenshot of the History Graph card, when the sensor has a `unit_of_measurement` defined.
</p>

> If the sensor does not have or requires the `unit_of_measurement` to be overridden, see [Manual Customization](/docs/configuration/customizing-devices/#manual-customization)

Reviewer: ```suggestion
<div class='note'>
If the sensor does not have a `unit_of_measurement` and should, see [Manual Customization](/docs/configuration/customizing-devices/#manual-customization)
</div>
```
I would drop the "requires to be overridden" part. Then it brings up a new unanswered question "when does this need to be overridden"? Also a sensor shouldn't have an incorrect `unit_of_measurement` that needs to be overridden. If one does it should be reported as an issue to the integration author.

Also think the note class is more fitting here. Otherwise its a bit of a strange first sentence in this article.
